src/data/full-quiz.js

```diff
@@ -1,4 +1,48 @@
 const fullQuiz = [
+  {
+    Question:
+      "A variable that has been declared, but has not been assigned a value is _______________",
+    Answer: "undefined",
+    Distractor1: "unknown",
+    Distractor2: "unassigned",
+    Distractor3: "pending",
+    Explanation:
+    "A variable that has not been assigned a value is of type undefined",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined",
+  },
+  {
+    Question:
+      "A variable that has been declared and assigned a value of 'no value' is ___________________",
+    Answer: "null",
+    Distractor1: "non-value",
+    Distractor2: "empty",
+    Distractor3: "unreturned",
+    Explanation:
+    "Null value is assigned to a variable to represent that the variable exists, but has no type or value.",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null",
+  },
+  {
+    Question:
+      "What is the index of the first element in an array in JavaScript?",
+    Answer: "0",
+    Distractor1: "1",
+    Distractor2: "first",
+    Distractor3: "initial",
+    Explanation:
+    "JavaScript arrays are zero-indexed. The first element of an array is located at index 0.",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array",
+  },
+  {
+    Question:
+      "In a switch statement, the __________________ case executes if none of the options are true.",
+    Answer: "default",
+    Distractor1: "override",
+    Distractor2: "else",
+    Distractor3: "alert",
+    Explanation:
+    "A default clause is executed if the value of expression doesn't match any of the case clauses.",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch",

```

Comment:
Whenever possible we like to use freeCodeCamp resources first before moving to MDN.

Proposed fix:
Link:"https://www.freecodecamp.org/news/javascript-switch-case-js-switch-statement-example/",